README.md

```diff
@@ -37,8 +37,14 @@ Once the shared library has been build, you will need to place it somewhere that
 * If you want to force a 32-bit build (I'm looking at you Excel 2016 on Mac), you can do:
 
     ``cmake .. -DREFPROP_32BIT=ON``
+    
+    --Note forcing a 32-bit build is no longer required for Office 365--
+
+* You have to use numpy <= 1.15.4 as 1.16+ will not build the .dylib file
```

Comment:
Really?  Can you please open an issue with the output for a build with homebrew gfortran and show the shell output?

Reply:
done. do you want me to close this pull request or leave it open?

Reply:
Let's close this one I think...